spanner/integration_test.go

```diff
@@ -386,6 +388,13 @@ func getInstanceConfig() string {
 	return os.Getenv("GCLOUD_TESTS_GOLANG_SPANNER_INSTANCE_CONFIG")
 }
 
+func getMultiplexEnableFlag() string {
+	if os.Getenv("GOOGLE_CLOUD_SPANNER_MULTIPLEXED_SESSIONS") == "true" {
+		return "true"
+	}
```

Comment:
Not very important, but this seems a bit strange. Why are we returning a standardized string from this method, instead of a bool?